.github/workflows/sv_pipeline_docker.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Docker Images
+
+on: 
+  push:
+    branches:
+      - master
+    paths:
+      - 'src/**'
```

Comment:
Should we also include `dockerfiles/**`?

Reply:
Agree, updated.